src/modules/launcher/PowerLauncher/LauncherControl.xaml

```diff
@@ -38,11 +38,21 @@
                                     </Style>
                                 </TextBlock.Style>
                             </TextBlock>
-                            <Border x:Name="border" Background="{TemplateBinding Background}" BorderThickness="{TemplateBinding BorderThickness}" BorderBrush="{TemplateBinding BorderBrush}" SnapsToDevicePixels="True">
-                                <ScrollViewer x:Name="PART_ContentHost" Background="{TemplateBinding Background}" Focusable="false" HorizontalScrollBarVisibility="Hidden" VerticalScrollBarVisibility="Hidden">
+                            <Border x:Name="border"
+                                    Background="{TemplateBinding Background}"
+                                    BorderThickness="{TemplateBinding BorderThickness}"
+                                    BorderBrush="{TemplateBinding DialogBorderBrush}"
```

Comment:
This is causing a compile time error. I think this should be `BorderBrush="{DynamicResource DialogBorderBrush}"`.

Reply:
@niels9001 fyi

Reply:
Fixed